models/KINOVA_merge.bioMod

```diff
@@ -479,6 +479,14 @@ segment part7
         -0.2155116207 -0.9764422143  0.010739928  -0.0752000001
         -0.9759714555  0.2157439188  0.0305660713  0.0291000003
          0.            0.            0.            1.
+    translations xyz
+    rotations xyz
+    ranges		-20.0	20.0
+	-2.0	4.0
+	-5.0	5.0
+	-pi/2     pi/2
+	-pi/2     pi/2
+	-pi/2     pi/2
```

Comment:
it should be another biomod, KINOVA_merge_with_6_dof_support.bioMod
and update the readme

Reply:
it was a mistake, there is no need to put 6 dof on the support in this model